dot/dot.py

```diff
@@ -54,10 +57,33 @@ def __init__(
         # additional attributes
         self.use_cam = (not use_video) and (not use_image)
 
+        # verify if camera exists, if _dot.use_cam is True
+        if self.use_cam:
+            if isinstance(target, int):
+                if not self.cam_exists(target):
+                    raise ValueError(f"Camera {target} does not exist.")
+            else:
+                raise ValueError("target must be an integer when using camera.")
+
         # create output folder
         if self.save_folder and not Path(self.save_folder).exists():
             Path(self.save_folder).mkdir(parents=True, exist_ok=True)
 
+    def cam_exists(self, target: int) -> bool:
```

Comment:
please move this into `camera_utils.py` . Some refactoring could be possible too. Ideally, we would only call `VideoCaptureAsync(target)` once. While we already call it here: https://github.com/sensity-ai/dot/blob/6d1a4a0d8bf98e69cc746ac79f7e38caca863d44/dot/commons/camera_utils.py#L54

Reply:
sure.